client/trino-jdbc/src/main/java/io/trino/jdbc/NonRegisteringTrinoDriver.java

```diff
@@ -90,8 +89,10 @@ public Connection connect(String url, Properties info)
     private Call.Factory instrumentClient(OkHttpClient client)
     {
         try {
-            OpenTelemetry openTelemetry = GlobalOpenTelemetry.get();
-            return OkHttpTelemetry.builder(openTelemetry).build().newCallFactory(client);
+            return OkHttpTelemetry
```

Comment:
should we log something in `catch` clause. How is user of library know that they need to setup something to get instrumetnation?

Reply:
There is no logging in the JDBC